domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java

```diff
@@ -179,7 +186,7 @@ public static Claim withDefaultCountyCourtJudgment() {
             .withCountyCourtJudgment(
                 SampleCountyCourtJudgment.builder()
                     .paymentOption(PaymentOption.IMMEDIATELY)
```

Comment:
pls change the paymentoption.IMMEDIATELY also as static import like you have done for DEFAULT

Reply:
done